apps/site/react_renderer/app.js

```diff
@@ -11,6 +11,20 @@ import ScheduleDirection from "../assets/ts/schedule/components/ScheduleDirectio
 import TripPlannerResults from "../assets/ts/trip-plan-results/components/TripPlannerResults";
 import MoreProjectsTable from "../assets/ts/projects/components/MoreProjectsTable";
 
+// log the process id when a process is started
+process.stdout.write(`node_logging node_process process=${process.pid}\n`);
+
+// log the memory usage each minute
+const memoryUsage = () => {
+  const used = process.memoryUsage().heapUsed / 1024 / 1024;
+  process.stdout.write(
+    `node_logging node_memory process=${process.pid} memory_in_mb=${Math.round(
```

Comment:
@meagonqz as a further example, the helper I'm thinking about here would here be invoked `log("node_memory", {process: process.pid, memory_in_mb: Math.round(used * 100) / 100})`